docs/api/serviceorchestration/orchestration-generic-http.md

```diff
@@ -0,0 +1,241 @@
+# orchestration IDD
```

Comment:
this is the IDD for the dynamic orchestration. File name, document title and content should express this.